float32 thrust_value
---
string status